code/game/machinery/doors/airlock.dm

```diff
@@ -1065,11 +1068,13 @@ About the new airlock wires panel:
 			to_chat(user, SPAN_NOTICE("Your [W.name] doesn't have enough fuel."))
 			return
 		playsound(src, 'sound/items/Welder.ogg', 50, 1)
+		show_sound_effect(get_turf(src), null, "small")
```

Comment:
I'd rather you make the sound icons a `#define` instead of just `"small"` etc
`#define SOUND_ICON_SMALL "small"`